tools/ipa_packageset_builder.py

```diff
@@ -0,0 +1,82 @@
+from jinja2 import Template
+import argparse
+from os import path
+from os.path import isfile
+DIR_PATH_TO_TEMPATES = "languages"
+DIR_PATH_TO_PACKAGES = "libraries"
+RUNNER_NAME = "Dockerfile.runner.j2"
+LANGSERVER_VERSION = "ce3f89098fddfaf4db8639a97ce3c0317abbd971"
+LANGSERVER_IMAGE ="algorithmiahq/langserver:{}".format(LANGSERVER_VERSION)
+RUNNER_PATH = path.join(DIR_PATH_TO_TEMPATES, RUNNER_NAME)
+
+class Package:
+    def __init__(self, package_name, install_script, dockerfile_path, base_image):
+        self.script = install_script
+        if dockerfile_path:
+            self.dockerfile = get_dockerfile_as_string(dockerfile_path)
+        else:
+            raise Exception('dockerfile path not available for package {}'.format(package_name))
+        if base_image:
```

Comment:
i don't think packages need the `base_image`?

Reply:
the template doesn't have logic to remove the `FROM {{base_image}}` like an if check, so I assumed it was mandatory.

Reply:
it needs to be in the context but it's not a property on each package that is referenced (it seems to properly be in the context down below)